modules/microvm/virtualization/microvm/netvm.nix

```diff
@@ -9,6 +9,14 @@
   configHost = config;
   vmName = "net-vm";
   macAddress = "02:00:00:01:01:01";
+
+  isGuiVmEnabled = config.ghaf.virtualization.microvm.guivm.enable;
```

Comment:
why is there a dependency on the ui vm?

Reply:
For example, AGX Orig doesn´t have GuiVM -- so we don´t have "waypipe" ssh-keys -- we don´t need to mount the keys to NetVM from the host and configure ssh client

Reply:
As long as NetVM uses ssh keys from GuiVM -- it is dependent on GuiVM